File: app/src/main/java/com/kushcabbage/friday_android/exceptions/RequestsExceededException.java
package com.kushcabbage.friday_android.exceptions;

import com.kushcabbage.friday_android.IModifyUI;

public class RequestsExceededException extends Exception {


    public void printToScreen(IModifyUI modifyUI) {
        modifyUI.showException("Accuweather requests exceeded");

    }
}

File: app/src/main/java/com/kushcabbage/friday_android/exceptions/RequestsExceededException.kt
package com.kushcabbage.friday_android.exceptions

import com.kushcabbage.friday_android.IModifyUI

class RequestsExceededException : Exception() {


    fun printToScreen(modifyUI: IModifyUI) {
        modifyUI.showException("Accuweather requests exceeded")

    }

    fun printStackTrace(modifyUI: IModifyUI) {
        super.printStackTrace()

    }
}
